src/SFA.DAS.Forecasting.Azure.Resources/deployments/function-app-resources.json

```diff
@@ -116,14 +131,34 @@
               "name": "Commitments-ClientToken",
               "value": "[parameters('CommitmentsClientToken')]"
             },
+            {
+              "name": "Commitments-ApiBaseUrl",
```

Comment:
Can the settings names all be renamed to the parameter name eg "CommitmentsV1ApiBaseUrl". It won't be consistent with the old JWT authentication settings but they could also be changed too.

Reply:
Ok